unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarShape.cs

```diff
@@ -229,14 +229,16 @@ public override void Cleanup()
             }
         }
 
-        private IEnumerator SetFacialFeaturesVisibleRoutine()
+        private IEnumerator CheckDistanceToPlayerRoutine()
         {
             while (true)
             {
                 yield return WaitForSecondsCache.Get(DISABLE_FACIAL_FEATURES_DISTANCE_DELAY);
                 Vector3 position = lastAvatarPosition ?? (entity.gameObject.transform.position + CommonScriptableObjects.worldOffset);
                 float distanceToPlayer = Vector3.Distance(CommonScriptableObjects.playerWorldPosition, position);
-                avatarRenderer.SetFacialFeaturesVisible(distanceToPlayer <= DISABLE_FACIAL_FEATURES_DISTANCE);
+                bool isNear = distanceToPlayer <= DISABLE_FACIAL_FEATURES_DISTANCE;
```

Comment:
Here we should check if the avatar is loaded to avoid weirdness.

Reply:
I dont agree with that. The AvatarShape just rely the actual enabling/disabling to the `AvatarRenderer`. 
If the avatar is not loaded yet we still have to call the AvatarRenderer so it has into account the state of the SSAO/FacialFeatures for the moment is ready.